src/view/ThemeManager.js

```diff
@@ -358,13 +402,13 @@ define(function (require, exports, module) {
     EditorManager.on("activeEditorChange", function () {
         refresh();
     });
-
     
     EventDispatcher.makeEventDispatcher(exports);
     
     exports.refresh         = refresh;
     exports.loadFile        = loadFile;
     exports.loadPackage     = loadPackage;
+    exports.addTheme        = addTheme;
```

Comment:
very small nit: Is there a reason why this is added here? It's strange that it wasn't appended to the end or alphabetically placed.